src/main/java/net/minecraftforge/items/wrapper/SidedInvWrapper.java

```diff
@@ -142,6 +145,17 @@ public ItemStack insertItem(int slot, @NotNull ItemStack stack, boolean simulate
                 return stack;
 
             m = Math.min(stack.getMaxStackSize(), getSlotLimit(slot));
+
+            // A few special cases to account for canPlaceItem implementations attempting to limit specific inputs to 1,
+            // by returning false if there's already a contained item. This doesn't work with modded inserted sizes > 1.
+            // - Limit buckets to 1 in furnace fuel inputs.
+            // - Limit brewing stand "bottle" inputs to 1.
+            if (inv instanceof AbstractFurnaceBlockEntity && getSlot(inv, slot, side) == 1 && stack.is(Items.BUCKET)
+                    || inv instanceof BrewingStandBlockEntity && getSlot(inv, slot, side) < 3)
```

Comment:
Since you've filed issue #9178 on this, we should probably add the issue number here for posterity.

Also, in the interest of ensuring maximum clarity, please surround the two conditions of the `||` with parentheses.